components/match2/wikis/counterstrike/match_summary.lua

```diff
@@ -0,0 +1,506 @@
+	local footer = MatchSummary.Footer()
+
+	local dot = '<b>·</b>'
+
+	local function createFooterLink(icon, url, label, index)
+		local _icon = icon
+		if icon == 'stats' then
+			_icon = index ~= 0 and 'Match Info Stats' .. index .. '.png' or 'Match Info Stats.png'
+		end
+		if index > 0 then
+			label = label .. ' for Game ' .. index
+		end
+
+		return '[[FILE:' .. _icon .. '|link=' .. url .. '|15px|' .. label .. '|alt=' .. url .. ']]'
+	end
+
+	-- Match vod
+	if not Logic.isEmpty(match.vod) then
+		footer:addElement(VodLink.display{
+			vod = match.vod,
+			source = match.vod.url
+		})
+	end
+
+	-- Game Vods
+	for index, vod in pairs(vods) do
+		-- TODO: Darkmode VodIcons
+		footer:addElement(VodLink.display{
+			gamenum = index,
+			vod = vod,
+			source = vod.url
+		})
+	end
+
+	if Table.isNotEmpty(match.links) then
+		if Logic.isNotEmpty(vods) or match.vod then
+			footer:addElement(dot)
+		end
+	else
+		return footer
+	end
+
+	--- Platforms is used to keep the order of the links in footer
+	local platforms = mw.loadData('Module:MatchExternalLinks')
+	local links = match.links
+
+	local insertDotNext = false
+	local iconsInserted = 0
+
+	for _, platform in ipairs(platforms) do
+		if Logic.isNotEmpty(platform) then
+			local link = links[platform.name]
+			if link then
+				if insertDotNext then
+					insertDotNext = false
+					footer:addElement(dot)
+				end
+
+				local icon = platform.icon
+				local label = platform.label
+				local addGameLabel = platform.isMapStats and match.bestof and match.bestof > 1
+
+				for _, val in ipairs(link) do
+					footer:addElement(createFooterLink(icon, val[1], label,
+														addGameLabel and val[2] or 0))
+					iconsInserted = iconsInserted + 1
+				end
+
+				if platform.stats then
+					for _, site in ipairs(platform.stats) do
+						if links[site] then
+							footer:addElement(dot)
+							break
+						end
+					end
+				end
+			end
+		else
+			insertDotNext = iconsInserted > 0 and true or false
+			iconsInserted = 0
+		end
+	end
+
+	return footer
+end
+
+function CustomMatchSummary._createMap(game)
+	local row = MatchSummary.Row()
+	local extradata = game.extradata or {}
+
+	-- Score
+	local team1Score = Score():setLeft()
+	local team2Score = Score():setRight()
+
+	-- Score Team 1
+	team1Score:setMapScore(game.scores[1])
+
+	local t1sides = extradata['t1sides'] or {}
+	local t1halfs = extradata['t1halfs'] or {}
+	local t2halfs = extradata['t2halfs'] or {}
+
+	local d = #t1sides
+
+	if t1sides then
+		-- Insert team scores
+		for i,side in ipairs(t1sides) do
+			local oppositeSide = CustomMatchSummary._getOppositeSide(t1sides[d])
+			-- Team 1 scores inserted from 1 .. n
+			if math.fmod(i,2) == 1 then
+				team1Score:setFirstHalfScore(t1halfs[i], CustomMatchSummary._getSideColor(side))
+			else
+				team1Score:setSecondHalfScore(t1halfs[i], CustomMatchSummary._getSideColor(side))
+			end
+			-- Team 2 scores inserted from n .. 1
+			if math.fmod(d, 2) == 1 then
```

Comment:
```suggestion
			if math.fmod(d - i, 2) == 1 then
```
And removing the `d = d - 1`.

Reply:
`d = d- 1` is needed because for team2 we start adding scores from the last `t2halfs`element, this is needed to mirror team1 half scores.
![image](https://user-images.githubusercontent.com/43279191/182588920-1287a2c5-06d3-4bd4-a58d-51fb8daa14d6.png)